File: PipelineDocumentation/ini_templates_sample_data/loadData.m
%% Reading time traces (clean_tv, TimeVector...) - float64

fileName = fullfile('E:\Pipeline_Projects\database\2025\RBM\Met\clean_tv');
fid = fopen(fileName);
if fid >0
    tv = fread(fid,'float64');
    fclose(fid);
end
% converting time from Matlab datenum to Matlab datetime format
tv_dt = datetime(tv,'ConvertFrom','datenum');

%% Reading data traces - float32

fileName = fullfile('E:\Pipeline_Projects\database\2025\RBM\Met\MET_HMP_T_6m_Avg');
fid = fopen(fileName);
if fid >0
    TA_1_1_1 = fread(fid,'float32');
    fclose(fid);
end

File: PipelineDocumentation/ini_templates_sample_data/loadData.py
import numpy as np
import matplotlib.pyplot as plt
from datetime import datetime, timedelta

# Reading time traces (clean_tv, TimeVector...) - float64
file_path_tv = r"E:\Pipeline_Projects\database\2024\RBM\Met\clean_tv"
try:
    with open(file_path_tv, "rb") as f:
        tv = np.fromfile(f, dtype=np.float64)
except FileNotFoundError:
    tv = np.array([])

# Converting time from MATLAB datenum to Python datetime format
days = tv % 1
tv_dt = [datetime.fromordinal(int(date)) + timedelta(days=float(date % 1)) - timedelta(days=366) for date in tv]

# Reading data traces - float32
file_path_data = r"E:\Pipeline_Projects\database\2024\RBM\Met\MET_HMP_T_6m_Avg"
try:
    with open(file_path_data, "rb") as f:
        TA_1_1_1 = np.fromfile(f, dtype=np.float32)
except FileNotFoundError:
    TA_1_1_1 = np.array([])
